docs/api/covidcast-signals/nchs-mortality.md

```diff
@@ -0,0 +1,91 @@
+| `deaths_pneumonia_notflu_incidence_prop` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths, per 100,000 population |
+| `deaths_covid_and_pneumonia_notflu_incidence_num`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza |
+| `deaths_covid_and_pneumonia_notflu_incidence_prop`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza, per 100,000 population |
+|`deaths_pneumonia_or_flu_or_covid_incidence_num`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19|
+|`deaths_pneumonia_or_flu_or_covid_incidence_prop`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19, per 100,000 population|
+|`deaths_percent_of_expected`| The number of weekly new deaths for all causes in 2020 compared to the average number across the same week in 2017–2019|
+
+These signals are taken directly from [Table
+1](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm) without
+changes. National provisional death counts include deaths occurring within the
+50 states and the District of Columbia that have been received and coded as of
+the date specified during a given time period. The deaths are classified based
+on a new ICD-10 code. (Note that the classification is based on all the codes on
+the death certificate, not just the primary cause of death). The codes that are
+considered for each signals are described in detail
+[here](https://github.com/cmu-delphi/covidcast-indicators/blob/nchs_mortality/nchs_mortality/DETAILS.md#metrics-level-1-m1). We
+export the state-level data as-is in a weekly format.
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Geographical Exceptions
+
+New York City is considered as a special state in the NCHS Mortality data, but
+we don't consider NYC separately. The death counts for NYC are included in New
+York State in our reports.
+
+## Report Using Epiweeks
+
+We report the NCHS Mortality data in a weekly format (`time_type=week` \&
+`time_value=\{YYYYWW\}`, where `YYYYWW` refers to an epiweek). The CDC defines
+the [epiweek](https://wwwn.cdc.gov/nndss/document/MMWR_Week_overview.pdf) as
+seven days, from Sunday to Saturday. We check the week-ending dates provided in
+the NCHS morality data and use Python package
+[epiweeks](https://pypi.org/project/epiweeks/) to convert them into epiweek
+format.
+
+## Lag and Backfill
+
+There is a lag in time between when the death occurred and when the death
+certificate is completed, submitted to NCHS, and processed for reporting
+purposes. The death counts for earlier weeks are continually revised and may
+increase or decrease as new and updated death certificate data are received from
+the states by NCHS. This delay can range from 1 week to 8 weeks or even more. We
+check for updates reported by NCHS every weekday but will report the signals
+weekly (on Monday). The data changes due to backfill made over the past week can
+be fetched from our API.
```

Comment:
We can add these two lines or not. I borrowed them from CDC website.

This delay can range from 1 week to 8 weeks or even more. Some states report deaths on a daily basis, while other states report deaths weekly or monthly. State vital record reporting may also be affected or delayed by COVID-19 related response activities which makes death counts not comparable across states. We check for updates reported by NCHS every weekday but will report the signals weekly (on Monday).